apps/examples/e2e/tests/test-shapes.spec.ts

```diff
@@ -73,17 +73,17 @@ test.describe('Shape Tools', () => {
 		for (const { tool } of otherTools) {
 			// Find and click the button
 			if (!(await page.getByTestId(`tools.${tool}`).isVisible())) {
-				if (!(await page.getByTestId(`tools.more`).isVisible())) {
+				if (!(await page.getByTestId('tools.more.button').isVisible())) {
```

Comment:
We usually limited the nesting to two, kind of like we do for translations. Maybe `tools.more-button`? This locator also seems to be used a lot, maybe we can extract it into a variable?